server/sonar-server/src/main/java/org/sonar/server/duplication/ws/ShowResponseBuilder.java

```diff
@@ -122,7 +124,7 @@ private void writeFiles(ShowResponse.Builder response, Map<String, String> refBy
 
         ComponentDto project = getProject(file.projectUuid(), projectsByUuid, session);
         ComponentDto parentModule = getParentProject(file.getRootUuid(), parentModulesByUuid, session);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getRootUuid"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)